source/game_sa/StencilShadows.h

```diff
@@ -25,6 +25,8 @@ class CStencilShadows {
     static void RenderForVehicle(CStencilShadowObject*);
     static void RenderForObject(CStencilShadowObject*);
     static void Render(const CRGBA& color);
+    static void RenderBuffer(const CVector& pos);
+    static void SunSetPosiitonFromEntity(const CEntity* entity);
```

Comment:
that's my typo. sorry :D (you need to rename xrefs, if you have, yourself sadly)
```suggestion
    static void SunSetPositionFromEntity(const CEntity* entity);
```